logging/logrep_proto_1.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # site_ids=tucson&amp;message_text=wubba&amp;min_level=0&amp;max_level=999&amp;user_ids=spothier&amp;user_agents=LOVE
</pre><p>Is this API documented somewhere that we have found yet? or is this us querying and seeing the url that pops out? (if it is in an api that we found can we just note that here?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>